web/components/admin/TextField.tsx

```diff
@@ -128,25 +145,87 @@ export const TextField: FC<TextFieldProps> = ({
         </div>
       ) : null}
 
-      <div className="input-side">
-        <div className="input-group">
-          <Field
-            id={fieldId}
-            className={`field ${className} ${fieldId}`}
-            {...fieldProps}
-            {...(type !== TEXTFIELD_TYPE_NUMBER && { allowClear: true })}
-            placeholder={placeholder}
-            maxLength={maxLength}
-            onChange={handleChange}
-            onBlur={handleBlur}
-            onPressEnter={handlePressEnter}
-            disabled={disabled}
-            value={value as number | (readonly string[] & number)}
-          />
+      {fieldName !== 'adminPassword' ? (
```

Comment:
Same commnet about the hardcoded `adminPassword` check here, maybe a property can be added for the two different states.

Reply:
To fix this issue, I tried to create the `useState` boolean variable and `useEffect` function to listen to the variable `type` change for the input fields. However, when they were added the Regex Input Validators stopped working, and I could not find another way to resolve this issue. Currently, `type !== TEXTFIELD_TYPE_PASSWORD ` is being used to check if it is the password input field or not. Please feel free to suggest other possible solutions.